src/domain/event/__tests__/EventIsEnrolled.test.tsx

```diff
@@ -16,9 +22,81 @@ const emptyOccurrenceMock: MockedResponse = {
   },
 };
 
-const mocks: MockedResponse[] = [emptyOccurrenceMock];
+const occurrenceMock: OccurrenceQuery = {
+  occurrence: {
+    __typename: 'OccurrenceNode',
+    id: 'occurrence-id',
+    time: new Date(Date.now() + 48 * 60 * 60 * 1000),
+    event: {
+      __typename: 'EventNode',
+      id: 'event-id',
+      description: 'event description',
+      image: '',
+      imageAltText: null,
+      shortDescription: null,
+      name: null,
+      duration: null,
+      participantsPerInvite: EventParticipantsPerInvite.ChildAndGuardian,
+      eventGroup: null,
+    },
+    remainingCapacity: null,
+    childHasFreeSpotNotificationSubscription: null,
+    venue: {
+      __typename: 'VenueNode',
+      id: '',
+      name: null,
+      address: null,
+      accessibilityInfo: null,
+      arrivalInstructions: null,
+      additionalInfo: null,
+      wwwUrl: null,
+      wcAndFacilities: null,
+    },
+  },
+};
 
 it('renders snapshot correctly', () => {
-  const { container } = render(<EventIsEnrolled />, mocks);
+  const { container } = render(<EventIsEnrolled />, [emptyOccurrenceMock]);
   expect(container).toMatchSnapshot();
 });
+
+describe('unenrolment is possible only 48 before the occurrence', async () => {
```

Comment:
Does this work because of the ceiling rounding method used in `differenceInHours` in `shouldAllowUnenrolment`? I mean that the current time is not frozen here, and as the tests should take less than an hour the difference should evaluate to the same hour value?

Reply:
Now there is 10s cap that should ensure that (and no ceiling rounding).